src/main/java/mat/client/cqlworkspace/shared/EditorDisplay.java

```diff
@@ -0,0 +1,27 @@
+package mat.client.cqlworkspace.shared;
+
+import com.google.gwt.user.client.TakesValue;
+import com.google.gwt.user.client.ui.HasText;
+import com.google.gwt.user.client.ui.RequiresResize;
+import edu.ycp.cs.dh.acegwt.client.ace.AceAnnotationType;
+import edu.ycp.cs.dh.acegwt.client.ace.AceEditorCursorPosition;
+
+public interface EditorDisplay extends RequiresResize, HasText, TakesValue<String> {
```

Comment:
This interface is not directly used yet. I'm planning on gradually migrate all code from AceEditor class usage to that interface to decouple UI from the business logic in the Presenters.